autodocodec-api-usage/autodocodec-api-usage.cabal

```diff
@@ -290,6 +290,7 @@ library
     , genvalidity-aeson
     , genvalidity-scientific
     , genvalidity-text
+    , insert-ordered-containers
```

Comment:
Do we really need this extra dependency? It's not a big deal but still probably not necessary...

Reply:
Removed

Reply:
I've removed it.